extensions/wikia/UserProfilePageV3/models/FavoriteWikisModel.class.php

```diff
@@ -33,67 +39,67 @@ public function __construct( User $user ) {
 	}
 
 	/**
-	 * Gets top wikis from DB for devboxes from method UserIdentityBox::getTestData()
+	 * Gets top wikis from stats DB based on number of edits
 	 *
 	 * @param Integer|null $limit
 	 *
 	 * @return array
 	 */
-	private function getTopWikisFromDb( $limit = null ) {
-		global $wgCityId, $wgDevelEnvironment, $wgSpecialsDB;
-
-		if ( is_null( $limit ) ) {
-			$limit = self::MAX_FAV_WIKIS;
-		}
+	private function getTopWikisFromDb( $limit = self::MAX_FAV_WIKIS ) {
+		global $wgCityId, $wgSpecialsDB;
 
 		if ( empty( $wgCityId ) ) {
 			// staff/internal does not have stats db
 			$wikis = [];
-		} else if ( $wgDevelEnvironment ) {
```

Comment:
Do we not have to worry about this `$wgDevelEnvironment` anymore? Looks like it didn't make it into the new version.